Yona/fabric.apikey

```diff
@@ -0,0 +1 @@
+afec01627d129084ba6d47c21993cf1b0a9c7c70
```

Comment:
Here you go. The API key is now public on the internet. _*This file really really shouldn't be under version control!!*_

Can we generate a new API key?

Reply:
I am still wonder, how API key file get commited from slave machine as file is added into .gitignore. Just now I tried in my local machine(updated api file for testing), source tree didn't show me to commit. 
Any clues on this?

Reply:
It's to me more the opposite: if the file is tracked by Git, I would be surprised to see it ignores changes to it.